DOCS/man/input.rst

```diff
@@ -2044,21 +2054,21 @@ Property list
     Equivalent to ``vf-metadata/<filter-label>``, but for audio filters.
 
 ``idle-active``
-    Return ``yes`` if no file is loaded, but the player is staying around
+    Returns ``yes``/true if no file is loaded, but the player is staying around
```

Comment:
why keeping the yes here?

Reply:
Why would you remove it? The goal of this commit was to make it clear that boolean properties can be both true and yes depending on the context, so that e.g. someone who only uses the JSON IPC like the person who opened the issue doesn't get confused. Like I wrote in the commit message whenever "Whether..." didn't sound good I explicitly said yes/true so it is clear to the reader that it can have either of those values depending on how you retrieve it.

Reply:
that answers the question. btw i didn't look at the commit messages just the whole changes.